igvc_navigation/src/navigation_server/navigation_server.h

```diff
@@ -64,8 +64,8 @@ class NavigationServer
 
   ros::Time start_time_;
 
-  ros::Time time_of_last_get_path_;
-  ros::Duration time_between_get_path_ = ros::Duration(0.5);
+  ros::Rate replanning_rate_ = ros::Rate(1.0);
+  boost::mutex replanning_mtx_;
```

Comment:
The standard library already has a `std::mutex`, theres no need to use `boost::mutex`